pallets/gear/src/schedule.rs

```diff
@@ -776,7 +776,7 @@ impl Default for Limits {
 impl<T: Config> Default for InstructionWeights<T> {
     fn default() -> Self {
         Self {
-            version: 1200,
+            version: 1201,
```

Comment:
@breathx should I increase minor version here ? Instrumentation must be done after these changes and I wanna to be sure

Reply:
if you want, it will be bumped to next release version anyway

Reply:
good, I will leave like this than